detox/test/src/Screens/DatePickerScreen.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+import {
+  Text,
+  View,
+  StyleSheet,
+  DatePickerIOS
+} from 'react-native';
+
+export default class DatePickerScreen extends Component {
+
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    return (
+      <View style={{flex: 1, paddingTop: 20, justifyContent: 'center', alignItems: 'center'}}>
+        <DatePickerIOS testID = {'CustomDatePicker'}
```

Comment:
If there is `testID` why find by type in the test?

Reply:
UIPickerView have not accessibility ID, testID = {'CustomDatePicker'} need to be deleted